src/app/app.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+import { MSAL_INSTANCE, MsalService } from '@azure/msal-angular';
+import { IPublicClientApplication, PublicClientApplication } from '@azure/msal-browser';
+
+function MSALInstanceFactory(): IPublicClientApplication {
+  return new PublicClientApplication({
+    auth: {
+      clientId: '6226576d-37e9-49eb-b201-ec1eeb0029b6',
+      redirectUri: 'http://localhost:4200'
```

Comment:
These strings should be replaced with the placeholders like so: https://github.com/Azure-Samples/active-directory-javascript-singlepageapp-angular/blob/master/src/app/app.module.ts#L36-L38

Reply:
Ah, whoops, this is a test file, nvm.